x-pack/plugins/security_solution/public/flyout/document_details/shared/constants/local_storage.ts

```diff
@@ -9,4 +9,5 @@ export const FLYOUT_STORAGE_KEYS = {
   OVERVIEW_TAB_EXPANDED_SECTIONS:
     'securitySolution.documentDetailsFlyout.overviewSectionExpanded.v8.14',
   RIGHT_PANEL_SELECTED_TABS: 'securitySolution.documentDetailsFlyout.rightPanel.selectedTabs.v8.14',
+  LEFT_PANEL_EXPANDED: 'securitySolution.documentDetailsFlyout.leftPanelExpanded.v8.14',
```

Comment:
honestly when I implemented my first item to be storage I followed what was done on the Timeline guided tour (`securitySolution.timeline.newFeaturesTour.v8.12`). I thought that this represented when the functionality had been introduced 😆 in case one day we would want to remove it, we'd know how long it had been in use...

I'm updating the value from `8.14` to `8.15` to be consistent with the other 2 `FLYOUT_STORAGE_KEYS` values already there, unless you feel strongly about it?

Reply:
@PhilippeOberti , you are right.  I added `release number` to the timeline tours because I was thinking, we might want to remove the tour after couple of releases and wanted to make it explicit when it was actually added. But I am not sure local storage keys fall in the same category.

>I'm updating the value from 8.14 to 8.15 to be consistent with the other 2 FLYOUT_STORAGE_KEYS values already there, unless you feel strongly about it?

I think that because of this change, user will lose their already stored values in `8.14` and to avoid that we will have to [write a migration](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/timelines/containers/local_storage/index.tsx#L94).  

For example,

In `8.14`. user has `securitySolution.documentDetailsFlyout.rightPanel.selectedTabs.v8.14` saved in the local storage with values `['overview']`. 

Now they upgrade to `8.15`. We have updated the key to include `v8.15` which will cause us to ignore their previously stored values resulting in bad UX.

Does that make sense?

Reply:
> I think that because of this change, user will lose their already stored values in 8.14 and to avoid that we will have to [write a migration](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/timelines/containers/local_storage/index.tsx#L94).

I'm confused. How could they loose anything? I only modified the value added in this PR. This is a new functionality, it didn't exist before. This shouldn't have any impact on upgrades, or am I missing something?

Reply:
May I mistunderstood your below statement. I though you are updating the values of existing keys as well.

> I'm updating the value from 8.14 to 8.15 to be consistent with the other 2 FLYOUT_STORAGE_KEYS

Thanks for clarification.

Reply:
oooh I see now how my sentence was confusing! Just to be clear: I'm updating the added value to have 8.15, to reflect the logic I used with the other 2 values (which have 8.14), which is which release the functionality was release in :)

Sorry about that!